pages/[prefix]/drops/[id].vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <UnlockableCollectionBanner :drop="drop" />
```

Comment:
I can see that we don't need anymore to wrap template into one single div, is it because of the eslint rules?

Reply:
There are several `elements` wrapped by the `template`.
![image](https://github.com/kodadot/nft-gallery/assets/31397967/f39edafd-c939-46d8-bc90-1e90f79bd3e5)

Reply:
yes this is new, before it was throwing warning